Check/DoctrineMongoDb.php

```diff
@@ -19,18 +21,36 @@ public function __construct(ManagerRegistry $registry, $connectionName = null)
 
     public function check()
     {
-        $connection = $this->manager->getConnection();
-        $connection->connect();
-        
-        if ($connection->isConnected()) {
-            return new Success();
+        $connection = $this->manager->getConnection($this->connectionName);
+
+        // Using "mongo" PHP extension
+        if (\method_exists($connection, 'connect')) {
```

Comment:
one last question, why are you not using an instanceof check here?

Reply:
No special reason, I could have used `instanceof` instead of `method_exists`.

In this case, I would prefer `method_exists` to be sure the called method exists, no matter the class. But maybe `is_callable` would be preferable, what do you think?

Reply:
hmm I would have leaned to instanceof .. but anyway explanation works for me .. merging.